src/nvim/buffer_updates.c

```diff
@@ -180,7 +180,7 @@ void buf_updates_send_changes(buf_T *buf,
   // change notifications are so frequent that many dead channels will be
   // cleared up quickly.
   if (badchannelid != 0) {
-    ELOG("Disabling buffer updates for dead channel %llu", badchannelid);
+    ELOG("Disabling buffer updates for dead channel %lu", badchannelid);
```

Comment:
I think we normally use `%zu` for `uint64_t` (and this is compatible with MSVC)

Reply:
Okay, will change that!

Reply:
Yeah, `%zu` would be incorrect on 32-bit platforms.  `"%" PRIu64` is the correct fix.